tests/test_network_availability.py

```diff
@@ -0,0 +1,32 @@
+from distutils.log import error
+import json
+from substrateinterface import SubstrateInterface
+
+def create_connection(url):
```

Comment:
We can setup a CI check with this actually

Reply:
Yes, that's right, I thought do that later with a little bit more complex checks.